examples/omega2/led.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+from os import uname
+from socket import gethostname
+from time import sleep
+import sys
+import cloud4rpi
+import omega2
+
+# Put your device token here. To get the token,
+# sign up at https://cloud4rpi.io and create a device.
+DEVICE_TOKEN = '__YOUR_DEVICE_TOKEN__'
+
+DATA_SENDING_INTERVAL = 30  # seconds
+
+
+def main():
+    o2 = omega2.Omega2()
+
+    # Put variable declarations here
+    variables = {
+        'Omega LED': {
+            'type': 'bool',
+            'value': False,
+            'bind': o2.led_control
+        }
+    }
+
+    # Put system data declarations here
+    diagnostics = {
```

Comment:
OK, If it's about **OS Name** and **IPAddress**, I'll make it consistent with raspberrypi examples. In the new PR, I'm going to change the raspberrypi examples